server/index.js

```diff
@@ -40,10 +43,14 @@ if (process.env.LOG_LEVEL) {
   Hull.logger.transports.console.level = process.env.LOG_LEVEL;
 }
 
-Server({
+const options = {
   Hull,
   hostSecret: process.env.SECRET || "1234",
   devMode: process.env.NODE_ENV === "development",
   port: process.env.PORT || 8082,
   onMetric
-});
+};
+
+const app = express();
+
+server(app, options);
```

Comment:
we miss `connector.startApp()` here, and let's move `connector.setupApp()` here too